.github/workflows/selenium-noIngestion-tests.yml

```diff
@@ -26,6 +26,9 @@ on:
       - 'openmetadata-ui/src/main/resources/ui/**'
       - 'openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pagesWithoutData/**'
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
We need to use fallback value here as there are multiple events that this CI is listening to.
Ref - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value